vocabularies/ConflictResponse.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$Version": "4.01",
+    "$Reference": {
+        "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.json": {
+            "$Include": [
+                {
+                    "$Namespace": "Org.OData.Core.V1",
+                    "$Alias": "Core"
+                }
+            ]
+        },
+        "https://sap.github.io/odata-vocabularies/vocabularies/Common.json": {
+            "$Include": [
+                {
+                    "$Namespace": "com.sap.vocabularies.Common.v1",
+                    "$Alias": "Common"
+                }
+            ]
+        }
+    },
+    "com.sap.vocabularies.ConflictResponse.v1": {
```

Comment:
Tun wir uns mit einem so speziellen Namen einen Gefallen?

Reply:
Other suggestions are welcome.

Reply:
What about ErrorResponse?

Reply:
How about UserInteraction as the vocabulary name?

Reply:
ErrorResponse solves a more general problem: How can we embed structured data into an error response. Navigation properties annotated with UI.UserInteraction are just one example for this.

Reply:
UserInteraction sounds more generic to me. The question is really where we are heading here. Is this a vocabulary specifically for the user interaction purpose we are discussing today? Or will this rather be a Common-4.01 vocabulary for everything that does not fit into Common?

Reply:
It is a vocabulary that helps us embed structured data into error responses because we do not want to use 409 Conflict responses for that.

Reply:
Instance annotations for error responses are already in the Common vocabulary, so using another "Common" vocabulary seems appropriate:
- Common-4.01
- CommonPlus
- Common2
- ...

Reply:
Common_401 (hyphens and dots are not allowed in simple identifiers)?